packages/airnode-deployer/terraform/gcp/main.tf

```diff
@@ -56,18 +56,20 @@ module "run" {
 module "startCoordinator" {
   source = "./modules/function"
 
-  name                  = "${local.name_prefix}-startCoordinator"
-  entry_point           = "startCoordinator"
-  source_dir            = var.handler_dir
-  memory_size           = 512
-  timeout               = 65
-  configuration_file    = var.configuration_file
-  secrets_file          = var.secrets_file
-  region                = var.gcp_region
-  project               = var.gcp_project
-  schedule_interval     = 1
-  max_instances         = var.disable_concurrency_reservation ? null : 1
-  invoke_targets        = [module.run.function_name]
+  name               = "${local.name_prefix}-startCoordinator"
+  entry_point        = "startCoordinator"
+  source_dir         = var.handler_dir
+  memory_size        = 512
+  timeout            = 65
+  configuration_file = var.configuration_file
+  secrets_file       = var.secrets_file
+  region             = var.gcp_region
+  project            = var.gcp_project
+  schedule_interval  = 1
+  max_instances      = var.disable_concurrency_reservation ? null : 1
+  invoke_targets = {
```

Comment:
TBH: This formatting looks broken

Reply:
I agree. But it's how the `terraform fmt` thinks it's best :man_shrugging: I can add an empty line above `invoke_targets` if that helps.

Reply:
I don't care that much, it's just that it formatted the whole section for nothing (nothign changed there)